scripts/buildWebtypes.js

```diff
@@ -0,0 +1,235 @@
+const path = require('path');
+const fs = require('fs');
+const os = require('os');
+const { execSync } = require('child_process');
+
+const PLAIN_WEB_TYPES_FILE = 'web-types.json';
+const LIT_WEB_TYPES_FILE = 'web-types.lit.json';
+
+const blacklistedPackages = [
+  /^vaadin-/,
+  /^component-base/,
+  /^field-base/,
+  /^field-highlighter/,
+  /^icons/,
+  /^input-container/,
+  /^lit-renderer/,
+  /^polymer-legacy-adapter/,
+];
+
+// Additional attributes that will be added to all components
+const additionalAttributes = [
+  // Theme attribute is not properly declared at the moment
+  {
+    name: 'theme',
```

Comment:
It seems the `theme` attribute is not picked up by the Polymer Analyzer, it doesn't seem to be declared. Added `theme` to all components by default. That might not be accurate, but it should be good enough.